applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -733,21 +733,12 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 Configure the functionality of the Matter-Bridge device
 -------------------------------------------------------
 
-To enable the Matter smart plug functionality, run the following command:
+To enable the Matter smart plugin functionality, for example with :ref:`zephyr:nrf54h20dk_nrf54h20`, run the following command:
```

Comment:
How about using some simpler example? It's only about presenting the part required for smart plug, so -Dmatter_bridge_SNIPPET=onoff_plug, the rest is not needed. I would suggest the following: 
```suggestion
To enable the Matter smart plugin functionality, run the following command with *board_target* replaced with the board target name:
```